src/NuGet.Core/NuGet.Build.Tasks.Console/ExtensionMethods.cs

```diff
@@ -12,6 +12,18 @@ namespace NuGet.Build.Tasks.Console
     /// </summary>
     internal static class ExtensionMethods
     {
+        /// <summary>
+        /// Determines if the specified item has a property value that is equal to <see cref="bool.FalseString" />.
+        /// </summary>
+        /// <param name="item">The <see cref="IMSBuildItem" /> to get the metadata value from.</param>
+        /// <param name="name">The name of the property to get the value of.</param>
+        /// <param name="defaultValue">The default value to return if the specified metadata has no value.</param>
+        /// <returns><code>true</code> if the specified property value is equal to <see cref="bool.FalseString" />, otherwise <code>false</code>.</returns>
```

Comment:
Do we really need IsPropertyFalse if we already have `IsPropertyTrue` method? Why don't we just use `!IsPropertyTrue` instead?

Reply:
Great question.  As a best practice, you shouldn't introduce MSBuild properties if you can rely on a default.

For example:

```xml
<PropertyGroup>
  <IsSomethingEnabled>true</IsSomethingEnabled>
</PropertyGroup>
```

As long as you document this property, there's no reason to allocate a string to track the fact that its set to `true`.  Instead, you only have to check if the user turned it off by setting it to `false`.

We've added other properties with their values explicitly set to `true` and then check if the value is still `true`.  Going forward, I'd like us to adopt this pattern of not introducing a property and instead checking if the user has set it to `false`.  You also can't negate `IsPropertyTrue()` since we're just comparing strings and the only supported value is `false`, anything else is considered `true.

Reply:
Thank you for context.